src/components/ui/ProgressPanel.svelte

```diff
@@ -54,19 +54,68 @@
       default: 'View report'
     }),
     CONFORMANCE_LEVEL: $translate('WCAG.COMMON.CONFORMANCE_LEVEL'),
-    REPORT_FOR: $translate('UI.REPORT.REPORT_FOR')
+    REPORT_FOR: $translate('UI.REPORT.REPORT_FOR'),
+    REPORTED_ON: $translate('UI.REPORT.REPORTED_ON'),
+    REPORTED_ON_OF: $translate('UI.REPORT.REPORTED_ON_OF')
   };
 
-  $:  console.log('totalevaluated', totalEvaluated);
   $: conformanceTarget = $scopeStore['CONFORMANCE_TARGET'];
-  $: wgacVersion = $scopeStore['WCAG_VERSION'];
-  $: percentageEvaluated = (totalEvaluated / totalToEvaluate) * 100;
+  $: wcagVersion = $scopeStore['WCAG_VERSION'];
+  $: percentageTotalEvaluated = (totalEvaluated / totalToEvaluate) * 100;
 
   $: principles = [...new Set($wcag.map((a) => a.num.split('.')[0]))];
 
-  $: console.log($assertions);
+  $: upToWcagVersion = wcagVersion === "2.0" ?
+    ["2.0"] : ["2.0", "2.1"];
 
-  let totalEvaluated = 3;
+  $: filteredCriteria = {
+   1: $wcag.filter(item => item.num.startsWith("1.")).filter(isInScope) || {},
+   2: $wcag.filter(item => item.num.startsWith("2."))
+    .filter(isInScope) || {},
+   3: $wcag.filter(item => item.num.startsWith("3."))
+    .filter(isInScope) || {},
+   4: $wcag.filter(item => item.num.startsWith("4."))
+    .filter(isInScope) || {}
+  };
+
+  $: filteredAssertions = {
+   1: $assertions.filter(item => item.test.num.startsWith("1.")).filter(isEvaluated),
+   2: $assertions.filter(item => item.test.num.startsWith("2.")).filter(isEvaluated),
+   3: $assertions.filter(item => item.test.num.startsWith("3.")).filter(isEvaluated),
+   4: $assertions.filter(item => item.test.num.startsWith("4.")).filter(isEvaluated)
+  }
+
+  $: totalsPerPrinciple = {
+    1: {
+      "done": filteredAssertions[1].length,
+      "total": filteredCriteria[1].length,
+      "percentage": (filteredAssertions[1].length / filteredCriteria[1].length) * 100
+    },
+    2: {
+      "done": filteredAssertions[2].length,
+      "total": filteredCriteria[2].length,
+      "percentage": (filteredAssertions[2].length / filteredCriteria[2].length) * 100
+    },
+    3: {
+      "done": filteredAssertions[3].length,
+      "total": filteredCriteria[3].length,
+      "percentage": (filteredAssertions[3].length / filteredCriteria[3].length) * 100
+    },
+    4: {
+      "done": filteredAssertions[4].length,
+      "total": filteredCriteria[4].length,
+      "percentage": (filteredAssertions[4].length / filteredCriteria[4].length) * 100
+    },    
+  }
+
+  function isInScope(wcagSC) {
+    return  wcagSC.conformanceLevel.length <= conformanceTarget.length && conformanceTarget.length &&  upToWcagVersion.includes(wcagSC.version)
+  };
+
+  function isEvaluated(assertion) {
```

Comment:
This may be instead moved to Assertion class as method (isTested) in `stores/earl/assertionStore/models`.

Reply:
added to separate clean up task in https://github.com/w3c/wcag-em-report-tool/issues/411